lib/sidekiq/prometheus/exporter/templates/standard.erb

```diff
@@ -58,3 +58,7 @@ sidekiq_dead_jobs <%= format('%d', @overview_stats.dead_size) %>
 # TYPE sidekiq_queue_busy_workers gauge
 <% @workers_stats.by_queue.each do |queue, stats| %>sidekiq_queue_busy_workers{name="<%= queue %>"} <%= format('%i', stats.busy_workers) %>
 <% end %>
+# HELP sidekiq_host_processes The number of quiet/active processes running on the host.
+# TYPE sidekiq_host_processes gauge
+<% @workers_stats.by_host.each do |host, stats| %><% stats.each do |quiet, count| %>sidekiq_host_processes{host="<%= host %>",quiet="<%= quiet %>"} <%= format('%i', count) %>
+<% end %><% end %>
```

Comment:
Could you please apply metric rearrangements from the previous comment to every occurrence? Here and in the tests too

Reply:
I'm trying to push to your branch some updates, could you please tick the box which will allow me to push updates